docs/tsa-policy.md

```diff
@@ -0,0 +1,666 @@
+# Abstract
+
+   This document defines requirements for a baseline time-stamp policy
```

Comment:
Can you format this whole doc to be of a fixed width (80 chars?) to match the RFC

Reply:
Yes, good call.